ui/src/components/Dropzone.tsx

```diff
@@ -0,0 +1,121 @@
+import React, { useMemo, useEffect, useState } from 'react'
+import { useDropzone } from 'react-dropzone'
+
+const baseStyle = {
+  display: 'flex',
+  flexDirection: 'column',
+  alignItems: 'center',
+  justifyContent: 'center',
+  height: 300,
+  width: 300,
+  margin: 'auto',
+  borderWidth: 2,
+  borderRadius: 'var(--cardRadius)',
+  borderColor: '#eeeeee',
+  borderStyle: 'dashed',
+  backgroundColor: 'white',
+  color: '#bdbdbd',
+  outline: 'none',
+  transition: 'border .24s ease-in-out',
+}
+
+const activeStyle = {
+  borderColor: '#2196f3',
+}
+
+const acceptStyle = {
+  borderColor: '#00e676',
+}
+
+const rejectStyle = {
+  borderColor: '#ff1744',
+}
+
+const thumbsContainer = {
+  display: 'flex',
+  alignItems: 'center',
+  justifyContent: 'center',
+  flexDirection: 'row',
+  flexWrap: 'wrap',
+}
+
+const thumb = {
+  display: 'inline-flex',
+  marginBottom: 8,
+  width: 'auto',
+  height: 250,
+  padding: 4,
+  boxSizing: 'border-box',
+}

```

Comment:
does it make sense to standardize stuff with css vars for margins and padding here?